src/backend/lima.ts

```diff
@@ -1659,7 +1659,9 @@ CREDFWD_URL='http://${ hostIPAddr }:${ stateInfo.port }'
         const status = await this.status;
 
         if (defined(status) && status.status === 'Running') {
-          await this.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+          if (this.cfg?.enabled) {
+            await this.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+          }
```

Comment:
This part should be reverted — we always have the k3s service written to disk, just not always running.

Reply:
done